src/views/Guide.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-container>
+    <h1 style="margin:0;">Guide</h1>
+
+    <!--
+      TODO: make sections linkable
+
+      TODO: add a contents bar which:
+            * links to the guide sections
+            * is presented when space is available
+            * stays in a fixed position when scrolling (i.e. position: fixed)
+            * tracks scroll position
+    -->
+
+    <h2 id="task-and-job-states">
+      Task And Job States
+    </h2>
+    <!-- TODO document difference between tasks and jobs -->
+    <div id="card-grid">
+
+      <v-flex
+        md6
+        xs12
+      >
+        <v-card>
+          <v-card-title primary-title>
+            <p class="display-1 text--primary">Task State Vs Job State</p>
+          </v-card-title>
+          <v-card-text>
+            <table id="task-job-state-table">
+              <tr>
+                <td>Task</td>
+                <td></td>
+                <td>Job</td>
+              </tr>
+              <tr>
+                <td>
+                  <p>
+                    <!-- TODO - this better -->
+                    The part which is related to the
+                    workflow itself. These are states
+                    which can be triggered off of
+                  </p>
+                </td>
+                <td></td>
+                <td>
+                  <p>
+                    <!-- TODO - this better -->
+                    The status of a running job
+                    which may or may not be the
+                    same as the task status
+                  </p>
+                </td>
+              </tr>
+              <tr
+                v-bind:key="state"
+                v-for="state in states"
+              >
+                <td style="font-size: 2em;">
+                  <task :status="state" progress="33" />
+                </td>
+                <td>
+                  <span>{{ state }}</span>
+                </td>
+                <td style="font-size: 2em;">
+                  <job :status="state" />
+                </td>
+              </tr>
+            </table>
+          </v-card-text>
+        </v-card>
+      </v-flex>
+
+      <v-flex
+        md5
+        xs8
```

Comment:
```suggestion
        xs12
```
Actually this seems to fix it.